w_common/build.yaml

```diff
@@ -5,4 +5,6 @@ targets:
       # migrated to null-safety. At that point, it should be scoped only to
       # relevant files.
       mockito:mockBuilder:
-        enabled: false
+        generate_for:
+          include:
+            - "test/**.mg.dart"
```

Comment:
@evanweible-wf this was our attempt to apply your thoughts about mock generation. Take a look. 

I can confirm that adding this filter keeps mockBuilder from affecting general build times. With this generate_for directive it seems to take a similar amount of time as with the builder fully disabled. I expect as we add mocks to generate this is going to get pretty slow, but that's another problem.